src/mn_setup.c

```diff
@@ -1132,12 +1132,9 @@ static void DrawInstructions()
 
     const char *s = "";
 
-    if (flags & S_FUNC)
+    if (flags & S_ALTDESC)
```

Comment:
Sufficient to check for `item->desc` here? I mean, do we need the flag actually?

Reply:
Right, I found real bug here: https://github.com/fabiangreffrath/woof/pull/1861/commits/1692cb580ea45a0cd0caefdec6cb2f2d72e1a196